client/ayon_houdini/plugins/create/create_model.py

```diff
@@ -23,6 +23,7 @@ class CreateModel(plugin.HoudiniCreator):
     label = "Model"
```

Comment:
I know the model creator supports only abc at the moment and there is an intent to extend but what about updating the label to reflect the current status of the creator.
```suggestion
    label = "Model (Abc)"
```